sqlsynthgen/utils.py

```diff
@@ -44,22 +43,19 @@ def import_file(file_name: str) -> ModuleType:
 
 
 def download_table(table: Any, engine: Any) -> None:
-    """Download a Table and store it as a .csv file."""
-    csv_file_name = table.fullname + ".csv"
-    csv_file_path = Path(csv_file_name)
-    if csv_file_path.exists():
-        print(f"{str(csv_file_name)} already exists. Exiting...", file=stderr)
+    """Download a Table and store it as a .yaml file."""
+    yaml_file_name = table.fullname + ".yaml"
+    yaml_file_path = Path(yaml_file_name)
+    if yaml_file_path.exists():
+        print(f"{str(yaml_file_name)} already exists. Exiting...", file=stderr)
         sys.exit(1)
 
     stmt = select([table])
     with engine.connect() as conn:
-        result = list(conn.execute(stmt))
+        result = [dict(row.items()) for row in conn.execute(stmt)]
```

Comment:
I think we can get away without the items(), if we want

```suggestion
        result = [dict(row) for row in conn.execute(stmt)]
```